applications/ConvectionDiffusionApplication/custom_elements/symbolic_d_convection_diffusion_explicit.cpp

```diff
@@ -0,0 +1,1783 @@
+{
+    return Kratos::make_intrusive<SymbolicDConvectionDiffusionExplicit>(NewId, this->GetGeometry().Create(ThisNodes), pProperties);
+}
+
+/***********************************************************************************/
+
+template< unsigned int TDim, unsigned int TNumNodes >
+Element::Pointer SymbolicDConvectionDiffusionExplicit<TDim,TNumNodes>::Create(
+    IndexType NewId,
+    GeometryType::Pointer pGeom,
+    Properties::Pointer pProperties) const
+{
+    return Kratos::make_intrusive<SymbolicDConvectionDiffusionExplicit>(NewId, pGeom, pProperties);
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+template< unsigned int TDim, unsigned int TNumNodes >
+void SymbolicDConvectionDiffusionExplicit<TDim,TNumNodes>::CalculateLocalSystem(
+    MatrixType& rLeftHandSideMatrix,
+    VectorType& rRightHandSideVector,
+    const ProcessInfo& rCurrentProcessInfo)
+{
+    KRATOS_TRY;
+
+    const auto& r_geometry = this->GetGeometry();
+    const unsigned int local_size = r_geometry.size();
+
+    // Resize and intialize output
+    if (rLeftHandSideMatrix.size1() != local_size)
+        rLeftHandSideMatrix.resize(local_size, local_size, false);
+    if (rRightHandSideVector.size() != local_size)
+        rRightHandSideVector.resize(local_size, false);
+    noalias(rLeftHandSideMatrix) = ZeroMatrix(local_size, local_size);
+    noalias(rRightHandSideVector) = ZeroVector(local_size);
+
+    // Element variables
+    ElementVariables rVariables;
+    this->InitializeEulerianElement(rVariables,rCurrentProcessInfo);
+
+    // Compute tau
+    this->CalculateTau(rVariables);
+
+    // Execute RHS-LHS build
+    this->CalculateLocalSystemInternal(rVariables,rLeftHandSideMatrix,rRightHandSideVector);
+
+    KRATOS_CATCH("");
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+template< unsigned int TDim, unsigned int TNumNodes >
+void SymbolicDConvectionDiffusionExplicit<TDim,TNumNodes>::CalculateRightHandSide(
+    VectorType& rRightHandSideVector,
+    const ProcessInfo& rCurrentProcessInfo)
+{
+    KRATOS_TRY;
+
+    Matrix LeftHandSide;
+    this->CalculateLocalSystem(LeftHandSide,rRightHandSideVector,rCurrentProcessInfo);
+
+    KRATOS_CATCH("");
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+template< unsigned int TDim, unsigned int TNumNodes >
+void SymbolicDConvectionDiffusionExplicit<TDim,TNumNodes>::CalculateLeftHandSide(
+    MatrixType& rLeftHandSideMatrix,
+    const ProcessInfo& rCurrentProcessInfo)
+{
+    KRATOS_TRY;
+
+    VectorType RightHandSide;
+    this->CalculateLocalSystem(rLeftHandSideMatrix,RightHandSide,rCurrentProcessInfo);
```

Comment:
Same comment but for the RHS in this case.

Reply:
Indeed, you shouldn't implement the LHS in this element as it is intended to only work with an explicit scheme. Having said this, I'd throw an error within this function.

Reply:
I did implement also the LHS because this way I could solve also steady-state problems. I will apply your suggestion of removing the LHS computation, since the aim is to solve unsteady problems.